app/Console/Commands/FetchNativeBalances.php

```diff
@@ -56,18 +57,22 @@ private function handleAllWallets(Collection $networks): void
         $onlyOnline = $this->option('only-online');
 
         Wallet::query()
+            ->select(['id', 'address'])
             ->when($onlyOnline, fn ($query) => $query->online())
             ->when(! $onlyOnline, fn ($query) => $query->recentlyActive())
-            ->chunkById(100, function ($wallets) use ($networks) {
-                $wallets->each(fn ($wallet) => $this->handleWallet($wallet, $networks));
+            ->chunkById(25, function ($wallets) use ($networks) {
```

Comment:
maybe useful to make this into a constant and add the reasoning for using 25 here, otherwise someone might change that without knowing why it was done as we usually chunk by 100

Reply:
Fair enough. Fixed.